CONTRIBUTING.md

```diff
@@ -0,0 +1,119 @@
+# Contributing to GNOME-Nepal
```

Comment:
@n1rjal  can you exclude the documentation files from the pr  as separate pr is created for them are in the stage of review it might  create  some conflict among the merge files